templates/components/media/media.html.twig

```diff
@@ -1,4 +1,17 @@
-{%- set media_link = media_link|render_clean -%}
+{#What I want:#}
+{#if all the medias are empty:#}
+{#{%- if (media_img_src is empty) or (media_audio_src is empty) or (media_audio_src is empty) -%}#}
+{#no link:#}
+{#{%- set media_link = false -%}#}
+{#if one is entered: {% else %}#}
+{#{%- set media_link = media_link|render_clean -%}#}
+
+{%- if (media_img_src is empty) -%}
```

Comment:
```suggestion
{%- if media_img_src is empty and media_custom|render_clean('<img> <picture>') is empty -%}
```

Reply:
@jenbreese try my suggestion above and let me know if that works.

Reply:
@pookmish Cool am trying now.

Reply:
I removed the `<iframe>` part since we wouldn't want a link around an iframe.

Reply:
that worked for me.